menas/src/main/scala/za/co/absa/enceladus/menas/models/rest/errors/RequestTimeoutExpiredError.scala

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
```

Comment:
How did this pass the checks? It is failing my local build on scala style issue. @yruslan

Reply:
Ha maybe this is the reason my release build fails. 🤔 
I ran in twice during CQC, but obviously wasn't able to investigate deeper.